subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/Report.kt

```diff
@@ -44,6 +45,8 @@ interface Report {
      */
     fun addHtml(label: String, content: String)
 
+    fun addImageSynchronously(file: File): String
```

Comment:
And I consider it's a wrong place for that function because it uses only remote storage